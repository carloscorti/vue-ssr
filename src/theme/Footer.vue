<template>
  <footer class="footer">
    <div class="container">
      <div class="content has-text-centered">
        Check source code
        <a :href="gitSrc" target="_blank">GitHub</a>
      </div>
    </div>
  </footer>
</template>

<script>
export default {
  name: 'Footer',
  data() {
    return {
      gitSrc: 'https://github.com/carloscorti/vue-ssr'
    };
  }
};
</script>
