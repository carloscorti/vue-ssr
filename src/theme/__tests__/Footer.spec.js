import { shallowMount } from '@vue/test-utils';
import Footer from '../Footer.vue';

describe('Footer', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Footer);
    expect(wrapper).toMatchSnapshot();
  });
});
