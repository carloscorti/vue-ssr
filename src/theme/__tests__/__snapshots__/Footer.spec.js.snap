// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer class="footer">
  <div class="container">
    <div class="content has-text-centered">
      Check source code
      <a href="https://github.com/carloscorti/vue-ssr" target="_blank">GitHub</a>
    </div>
  </div>
</footer>
`;
